maven-core/src/main/java/org/apache/maven/plugin/DefaultPluginDescriptorCache.java

```diff
@@ -19,10 +19,7 @@
  * under the License.
  */
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
we do not use star import

Reply:
thanks for review.I have modified the code as your suggestion.